static/js/init.js

```diff
@@ -3,6 +3,7 @@
 /* global require, window, document */
 
 var $ = require('jquery');
+var keyboard = require('keyboardjs');
```

Comment:
Do we really need a whole library if we're just going to use a mapping of keys to key codes? Are we going to be using this library elsewhere for more in depth functionality?